woodwork/table_accessor.py

```diff
@@ -208,13 +215,11 @@ def __setitem__(self, col_name, column):
         if column.ww._schema is None:
             column = init_series(column, use_standard_tags=self.use_standard_tags)
         elif self.use_standard_tags and not column.ww.use_standard_tags:
-            column.ww._schema['semantic_tags'] |= column.ww.logical_type.standard_tags
+            column.ww.add_semantic_tags(column.ww.logical_type.standard_tags)
             message = StandardTagsChangedWarning().get_warning_message(self.use_standard_tags, col_name)
             warnings.warn(message, StandardTagsChangedWarning)
         elif not self.use_standard_tags and column.ww.use_standard_tags:
-            column.ww._schema['semantic_tags'] -= column.ww.logical_type.standard_tags
-            message = StandardTagsChangedWarning().get_warning_message(self.use_standard_tags, col_name)
-            warnings.warn(message, StandardTagsChangedWarning)
+            column.ww.remove_semantic_tags(column.ww.logical_type.standard_tags)
```

Comment:
The augmented assignment statements here and in the block above were causing issues now that we have semantic tags as an actual attribute instead of a dictionary element: `column.ww._schema.semantic_tags`. 

I figured we could make use of our WoodworkColumnAccessor operations here, and because `remove_semantic_tags` issues a `StandardTagsChangedWarning` when removing standard tags, it let us remove those lines here.